vars/standardJavaDeploy.groovy

```diff
@@ -57,7 +57,13 @@ def call(config) {
         stage('Build docker image') {
             container('docker') {
                 docker.withRegistry(config.registryUrl, config.registryCredentialsId) {
-                    sh "docker build -t ${dockerImage} --build-arg SOURCE_VERSION=${scmInfo.GIT_COMMIT} ."
+                    if (config.subModuleName != null) {
+                        dir("${env.WORKSPACE}/${config.subModuleName}") {
+                            sh "docker build -t ${dockerImage} --build-arg SOURCE_VERSION=${scmInfo.GIT_COMMIT} ."
```

Comment:
Perhaps make the script a variable so that it's not duplicated

Reply:
Thanks. Fixed.